### ISSUE TITLE
anime style? 

### ISSUE THREAD

**Issue by lorrp1** [AceSix/Knowledge_Transfer#1]
have you tried it to create anime style picture?

**Comment by AceSix** [comment]
Yes, we have. You can find results of anime style transfer in our paper(Fig.8), which can be accessed by the link we give in this repo.
